src/translate.py

```diff
@@ -3704,18 +3976,31 @@ def process_instr(instr: Instruction) -> None:
                 branch_condition = cond_bit.negated()
 
         elif mnemonic in arch.instrs_jumps:
-            assert mnemonic == "jr"
-            if args.reg_ref(0) == arch.return_address_reg:
-                # Return from the function.
+            if mnemonic == "b":
+                if isinstance(args.raw_arg(0), AsmGlobalSymbol):
+                    # Unconditional jump
+                    pass
```

Comment:
Remnant of tail call pattern? (mips puts `b` in instrs_ignore)

Reply:
Yeah I think so, I was able to put `b` in instrs_ignore for PPC too now.